thirdeye-ui/src/app/pages/welcome-landing-page/welcome-landing-page.component.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Grid, Typography } from "@material-ui/core";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { LoadingErrorStateSwitch } from "../../components/page-states/loading-error-state-switch/loading-error-state-switch.component";
+import { WelcomeStepCard } from "../../components/welcome-landing-page/welcome-step-card/welcome-step-card.component";
+import { PageContentsCardV1, PageV1 } from "../../platform/components";
+import { DimensionV1 } from "../../platform/utils";
+import { ActionStatus } from "../../rest/actions.interfaces";
+import { getAllDatasets } from "../../rest/datasets/datasets.rest";
+import { getDataConfigurationCreatePath } from "../../utils/routes/routes.util";
+
+export const WelcomeLandingPage: FunctionComponent = () => {
+    const [status, setStatus] = useState<
+        typeof ActionStatus[keyof typeof ActionStatus]
+    >(ActionStatus.Initial);
+    const [hasDatasets, setHasDatasets] = useState<boolean>(false);
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        setStatus(ActionStatus.Working);
+        getAllDatasets()
+            .then((datasets) => {
+                if (datasets && datasets.length > 0) {
+                    setHasDatasets(true);
+                }
+                setStatus(ActionStatus.Done);
+            })
+            .catch(() => {
+                setStatus(ActionStatus.Error);
+            });
+    }, []);
```

Comment:
We could simplify this with `useGetDatasets`

Reply:
Implemented